### ISSUE TITLE
Please publish to the puppet forge :)

### ISSUE THREAD

**Issue by flypenguin** [nickschuch/puppet-rancher#3]
... it's a puppet module after all. (Using puppet blacksmith this is also quite easy, in case you didn't know that tool)

**Comment by nickschuch** [comment]
Ill get it up onto the forge :)

**Comment by SydOps** [comment]
+1

**Comment by alexharv074** [comment]
@nickschuch   You should be able to just do:

```
puppet module build
```

then upload the `.tar.gz` file to the Forge.

@petems also seems to have set up Puppet Blacksmith, although I haven't used it.

**Comment by petems** [comment]
Yeah, puppet blacksmith just uploads the tarball to the forge from the comand line, makes things more automated but you can do the same thing on the forge page itself :smile:

**Comment by alexharv074** [comment]
Automate all the things ! :-)

**Comment by nickschuch** [comment]
Yep, uploaded to the forge and pushed a change after I merged: https://github.com/nickschuch/puppet-rancher/commit/2cb4f1089ede1272555a9ac4cabc81cf568875e0

https://forge.puppetlabs.com/nick/rancher

Gonna check out puppet blacksmith!
